docs/pages/access-controls/guides/headless.mdx

```diff
@@ -178,6 +179,26 @@ alice@server01 $
   your local terminal.
 </Notice>
 
+## Optional: Teleport Connect
+
+Teleport Connect v13.3.1+ can also be used to approve headless logins.
+Teleport Connect will automatically detect the headless login attempt
```

Comment:
This comment doesn't apply to this PR, since it's a backport, but we should go in later and make sure that the product name is consistent across the docs. There's a mix of "headless login" and "Headless WebAuthn".

Reply:
Sorry, my backport did not include your suggestsions by mistake, fixed in https://github.com/gravitational/teleport/pull/30594